packages/competence/bin/static/fragments/frame-employee-management.html

```
@@ -236,6 +236,34 @@
+                                            <select class="ti-select" x-model="draft.personal.gender"
+                                                    x-bind:disabled="!isFieldEditable('personal.gender')">
+                                                <option value="" x-text-label="interface.employee-management.gender.none"></option>
+                                                <template x-for="value in options.genders" x-bind:key="value">
+                                                    <option x-bind:value="value" x-text="value"></option>
+                                                </template>
+                                            </select>
+                                        </div>
+
+                                        <div class="ti-form-row">
+                                            <label class="ti-field-label" x-text-label="interface.employee-management.form.position-name"></label>
+                                            <input class="ti-input" type="text" x-model="draft.career.positionName"
```

HTMLHint (1.9.2)

[warning] No matching [ label ] tag found.

(input-requires-label)